lib/errors/course_error_records.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Errors
+  module CourseErrorRecords
```

Comment:
Following the convention in the (limited) amount of non-Helper modules in the code base, consider naming this after what it does: `CourseUpdateErrorHandling`.

Reply:
Done!